<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskPanel</class>
 <widget class="QWidget" name="TaskPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>707</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Import Alignment</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="panel_widget" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0">
      <item>
       <layout class="QVBoxLayout" name="panel_layout" stretch="0,1">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Example Task Panel</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0">
          <item>
           <widget class="QPushButton" name="draw_rectangle_button">
            <property name="text">
             <string>Draw Rectangle</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="draw_circle_button">
            <property name="text">
             <string>Draw Circle</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="delete_object_button">
            <property name="text">
             <string>Delete Selected Object</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
